manifests/install.pp

```diff
@@ -3,7 +3,7 @@
   $ensure  = 'latest',
   ) {
 
-  package { 'scout':
+  package { '^scout':
```

Comment:
Its a puppet problem code problem either way. I'm not going to waste that much time on this (as I only thought of fixing this annoying but not impacting problem as a quick fix). This is only now failing build because we never had tests (and this doesn't test implementation at all, just that we copy out the expected result). Local vagrant testing on trusty is successful.